src/server.js

```diff
@@ -0,0 +1,94 @@
+import express from "express";
+import dotenv from "dotenv";
+import cors from "cors";
+import mongoose from "mongoose";
+
+import booksData from "../data/books.json";
+import { Book } from "./models/Book";
+
+dotenv.config();
+
+const mongoUrl = process.env.MONGO_URL;
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
+
+const port = process.env.PORT;
+const app = express();
+
+app.use(cors());
+app.use(express.json());
+app.use((req, res, next) => {
+  if (mongoose.connection.readyState !== 1) {
+    res.status(500).json({ message: "Internal server error" });
+  } else {
+    next();
```

Comment:
Your server is returning this error so probably your API can't connect to the database.

Reply:
The database connection is resolved. would you please chack it again?